azure/templates/log-analytics.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "logAnalyticsWorkspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Specify the name of the workspace."
```

Comment:
We we add an example, EG
"description": "Specify the name of the workspace. eg: faltrn-kv-law"

Reply:
This is an input parameter unfortunately "logAnalyticsWorkspaceName=${RESOURCE_NAME_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-log" which is what will need to be set if the enable diagnostics is set to true